src/lib/server/xlat_eval.c

```diff
@@ -285,23 +285,23 @@ static xlat_action_t xlat_eval_one_letter(TALLOC_CTX *ctx, fr_cursor_t *out, REQ
 
 	case 'c': /* Current epoch time seconds */
 	{
-		struct timeval now;
+		fr_time_t now;
 
-		gettimeofday(&now, NULL);
+		now = fr_time();
 
 		MEM(value = fr_value_box_alloc(ctx, FR_TYPE_UINT64, NULL, false));
-		value->datum.uint64 = (uint64_t)now.tv_sec;
+		value->datum.uint64 = (uint64_t)fr_time_to_sec(now);
```

Comment:
```fr_time_to_sec(fr_time())``` which lets you remove the curly brackets around that code block too...

Reply:
@arr2036 done